server_admin/topics/admin-cli.adoc

```diff
@@ -789,6 +789,28 @@ Use the following example to remove two roles defined on the client [command]`re
 $ kcadm.sh remove-roles -r demorealm --rname testrole --cclientid realm-management --rolename create-client --rolename view-users
 ----
 
+[discrete]
+==== Adding client roles to a group
+
+Use a dedicated [command]`add-roles` command that can be used for adding realm roles and client roles.
+
+The following example adds the roles defined on the client [command]`realm-management` - `create-client` role and the [command]`view-users` role to the [command]`Testgroup` group (via the [command]`--gname` option). The group can alternatively be specified by ID (via the [command]`--gid` option).
```

Comment:
Can you make it consistent with section "Group operations", please. Use group name "Group".
Would be also nice to reference this section, so user can easily find related command how to create groups.

Reply:
> Use group name "Group".

No problem, done.

> Would be also nice to reference this section

Hm, I'm not sure how to exactly do this, what are the steps?

Reply:
Ah, interesting syntax. Ok, I think I did it in 8d39bf17f0fbc059600db3c475c7a13c72ab495b